victor/src/style/properties/definitions.rs

```diff
@@ -14,6 +14,7 @@ properties! {
     }
 
     reset struct box_ {
+        @early position { "position", Position, initial = Position::Static }
```

Comment:
Note that this is not going to work. You need to keep the original display value without blockification for hypothetical box computation. I'd blockify in a separate style fixup phase rather than at compute time.